recipes/usalign/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -exo pipefail
+
+readonly PROGRAMS=(
+    qTMclust
+    USalign
+    TMalign
+    TMscore
+    MMalign
+    se
+    pdb2xyz
+    xyz_sfetch
+    pdb2fasta
+    pdb2ss
+    NWalign
+    HwRMSD
+    cif2pdb
+    pdbAtomName
+    addChainID
+)
+
+if [[ $(uname -s) == "Darwin" ]]; then
+    export CXX="${BUILD_PREFIX}/bin/clang++"
```

Comment:
Why is this needed ?
I guess it is similar to your other recipe - https://github.com/bioconda/bioconda-recipes/pull/50461

Reply:
On MacOS build, CXX env variable seems to not be set.

So I needed to set it explicitly.

Reply:
This is because of ` - {{ compiler("cxx") }}  # [linux]` - the c++ compiler is added only for Linux.

Reply:
I think what you say is right